client/signup/steps/domains/index.jsx

```diff
@@ -906,8 +934,18 @@ export class RenderDomainsStep extends Component {
 
 		const hasSearchedDomains = Array.isArray( this.props.step?.domainForm?.searchResults );
 
+		// Ensure we don't show the migration explainer to non-English locales until all translations are complete.
+		const showMigrationExplainer =
+			englishLocales.includes( this.props.localeSlug || '' ) ||
+			hasTranslation( 'Migrating an existing site?' );
+
 		return (
 			<div className="domains__domain-side-content-container">
+				{ showMigrationExplainer && (
+					<div className="domains__domain-side-content domains__domain-site-migration">
+						<ReskinSideExplainer type="site-migration" onClick={ this.handleSiteMigrationClick } />
+					</div>
+				) }
```

Comment:
@markbiek, we need to put this together as an A/B test rather than as a basic conditional shown to all users.

Based on the code in /client/lib/explat/README.md, the structure here should be a bit different.
```suggestion
				<Experiment
					name="calypso_signup_domains_step_migration_sidebar**"
					defaultExperience={ null }
					treatmentExperience={ (
						<div className="domains__domain-side-content domains__domain-site-migration">
							<ReskinSideExplainer type="site-migration" onClick={ this.handleSiteMigrationClick } />
						</div>
					) }
```

** Note that the name would depend on the name of the A/B test we create.